src/openexr.imageio/exroutput.cpp

```diff
@@ -799,22 +799,28 @@ struct ExrMeta {
         : oiioname(oiioname), exrname(exrname), exrtype(exrtype) {}
 };
 
+// Make our own type here because TypeDesc::TypeMatrix and friends may
+// not have been constructed yet. Initialization order fiasco, ick!
+static TypeDesc TypeMatrix (TypeDesc::FLOAT,TypeDesc::MATRIX44,0);
+static TypeDesc TypeTimeCode (TypeDesc::INT, TypeDesc::SCALAR, TypeDesc::TIMECODE, 2);
+static TypeDesc TypeKeyCode (TypeDesc::INT, TypeDesc::SCALAR, TypeDesc::KEYCODE, 7);
+
 static ExrMeta exr_meta_translation[] = {
     // Translate OIIO standard metadata names to OpenEXR standard names
-    ExrMeta ("worldtocamera", "worldToCamera", TypeDesc::TypeMatrix),
-    ExrMeta ("worldtoscreen", "worldToNDC", TypeDesc::TypeMatrix),
-    ExrMeta ("DateTime", "capDate", TypeDesc::TypeString),
-    ExrMeta ("ImageDescription", "comments", TypeDesc::TypeString),
-    ExrMeta ("description", "comments", TypeDesc::TypeString),
-    ExrMeta ("Copyright", "owner", TypeDesc::TypeString),
-    ExrMeta ("PixelAspectRatio", "pixelAspectRatio", TypeDesc::TypeFloat),
-    ExrMeta ("XResolution", "xDensity", TypeDesc::TypeFloat),
-    ExrMeta ("ExposureTime", "expTime", TypeDesc::TypeFloat),
-    ExrMeta ("FNumber", "aperture", TypeDesc::TypeFloat),
-    ExrMeta ("oiio:subimagename", "name", TypeDesc::TypeString),
-    ExrMeta ("openexr:dwaCompressionLevel", "dwaCompressionLevel", TypeDesc::TypeFloat),
-    ExrMeta ("smpte:TimeCode", "timeCode", TypeDesc::TypeTimeCode),
-    ExrMeta ("smpte:KeyCode", "keyCode", TypeDesc::TypeKeyCode),
+    ExrMeta ("worldtocamera", "worldToCamera", TypeMatrix),
+    ExrMeta ("worldtoscreen", "worldToNDC", TypeMatrix),
+    ExrMeta ("DateTime", "capDate", TypeDesc::STRING),
```

Comment:
What makes the `STRING` case safe but not the `TypeMatrix` case?

Its a shame we don't have `constexpr` available yet ...